docs/new/posts/PostgreSQL/postgresql-17.5.1-release.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2025-05-28
+description: >
+  Percona Distribution for PostgreSQL 17.5.1 has been released on May 28, 2025.
+authors: [Andriciuc]
+categories:
+  - PostgreSQL
+tags:
+  - Percona Distribution for PostgreSQL
+---
+
+# Percona Distribution for PostgreSQL 17.5.1 has been released
+
+<!-- more -->
+
+[Percona Distribution for PostgreSQL](https://docs.percona.com/postgresql/17/index.html){:target="_blank"} 17.5.1 has been released on May 28, 2025.
```

Comment:
I belive there's a mess with dates. pg_tde is part of the distro so the Distro cannot be released without it. And the dates say the opposite - distro is released on 28.05 while pg_tde - on 29.05